tests/micro/zephyr/test_zephyr.py

```diff
@@ -67,6 +71,7 @@ def _make_session(temp_dir, zephyr_board, west_cmd, mod, build_config):
             "west_cmd": west_cmd,
             "verbose": bool(build_config.get("debug")),
             "zephyr_board": zephyr_board,
+            "main_stack_size": stack_size,
```

Comment:
I think this is the first time an option is passed that is of type 'None'. So far usually what we do if an option is deemed `None` is to simply don't pass the option and then use the following in the server side:
```
if `options.get("main_stack_size"):
    ... options["main_stack_size"]
```
So I personally prefer to keep this way instead of passing an option with value "None".  If that form is adopted in this case it would be a matter of checking for stack_size after it's set accordingly and then check if it's None, appending (or not) the "main_stack_size" option to the options dict. @areusch wdyt?

Reply:
I agree. changed it.